tests/unit/test_etl.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from unittest.mock import patch
+
+from dcpquery.etl import load_links
+
+
+class TestETLHelpers(unittest.TestCase):
+    @patch('dcpquery.etl.logger.error')
+    def test_links_ignore_unknown_format(self, logger):
+        load_links([{1: 2}])
+        logger.assert_called_once()
```

Comment:
you could also use assertRaises here to check that a particular exception was raised. https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

Reply:
Good idea, but the exception is not raised in this frame. It's caught before reaching this frame.